src/plugins/controls/public/options_list/components/options_list_strings.ts

```diff
@@ -126,7 +119,13 @@ export const OptionsListStrings = {
       }),
     getSortDisabledTooltip: () =>
       i18n.translate('controls.optionsList.popover.sortDisabledTooltip', {
-        defaultMessage: 'Ignore sorting when “Show only selected” is true.',
+        defaultMessage: 'Sorting is ignored when “Show only selected” is true',
+      }),
+    getDocumentCountTooltip: (documentCount: number) =>
+      i18n.translate('controls.optionsList.popover.documentCountTooltip', {
+        defaultMessage:
+          'This value appears in {documentCount} {documentCount, plural, one {document} other {documents}}',
+        values: { documentCount },
```

Comment:
@elastic/kibana-docs This text is for the tooltip that shows up when hovering on the document count:

![Dec-21-2022 16-31-27](https://user-images.githubusercontent.com/8698078/209022942-e8ff0340-38bf-4d9d-a97c-1c3038c5b57e.gif)


Let me know if you think this wording is clear :+1:

Reply:
Text LGTM.  Our guideline is to end tooltips with a period.  However, none of the tooltip text in this PR does that.  For consistency within Lens,  we can let it go now and make that change in one global sweep.